canonical-web-and-design/partners.ubuntu.com
# Add redirects to canonical.com/partners

## Done

- Add non authenticated pages to redirect to canonica.com/partners

## QA

1. Download this branch
2. `./run` the site
3. Go to http://0.0.0.0:8003/
4. try a few links and make sure they redirect properly
5. Try http://0.0.0.0:8003/admin and make sure you stay on the local site
